backend/code/m-booking/src/test/java/com/mbooking/repository/UserRepositoryTest.java

```diff
@@ -0,0 +1,64 @@
+package com.mbooking.repository;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.mbooking.model.Customer;
+import com.mbooking.model.User;
+
+@DataJpaTest
+@RunWith(SpringRunner.class)
+@ActiveProfiles("test_h2")
+public class UserRepositoryTest {
```

Comment:
Once again, consistency, name it UserRepositoryIntegrationTest

Reply:
ispravljeno